blog-post.html

```diff
@@ -1,12 +1,150 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <title>Blog Post</title>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
-  </head>
-  <body>
-    <section id="header"></section>
-    <section id="main"></section>
-  </body>
-</html>
+
+<head>
+  <title>Blog Post</title>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+  <link href="\styles\style.css" rel="stylesheet" />
+</head>
+
+<body>
+  <!----Top Page Header-->
+  <section id="header">
+    <header>
+      <a href="/index.html">
+        <h1>Post only</h1>
+      </a>
+      <h6>A page with just a blog post content</h6>
+    </header>
+    <br>
+    <br>
+    <section class="line-break">
+    </section><br>
+    <section class="line-break">
+    </section><br>
+
+    <centre>
+      <h3>This is a <b>Blog Post</b> Only Layout</h3>

```

Comment:
`<h3>` tags have an inherent property of being bold, if we wanted to style this correctly we can make this a `<span>` tag and adjust the css of it to make the text larger